src/main/kotlin/xta/game/creature/races/HorseRace.kt

```diff
@@ -0,0 +1,76 @@
+package xta.game.creature.races
+
+import xta.game.PlayerCharacter
+import xta.game.creature.Race
+import xta.game.creature.RacialStage
+import xta.game.creature.body.*
+import xta.game.stats.Stats
+
+object HorseRace : Race (18, "horse", 4) {
+
+    object STAGE_HALF_HORSE: RacialStage(this, "half equine-morph"
+    )
+
+    object STAGE_HORSE: RacialStage(this, "equine-morph"
+    )
+
+    override fun stageForScore(creature: PlayerCharacter, score: Int) = when {
+        score >= 7 -> STAGE_HORSE
+        score >= 4 -> STAGE_HALF_HORSE
+        else -> null
+    }
+
+    override fun basicScore(creature: PlayerCharacter): Int = with(creature) {
+        var score = 0
+        if (face.type == FaceType.HORSE)
+            score++
+        if (ears.type == EarType.HORSE)
+            score++
+        if (horns.type == HornType.UNICORN)
+            score = 0
+        if (tail.type == TailType.HORSE)
+            score++
+        if (lowerBody.type == LowerBodyType.HOOFED)
+            /*
+            TODO: Missing/comment-blocked Centaur LowerBodyType
+            || lowerBody.type == LowerBodyType.CENTAUR)
+            */
+            score++
+        if (countCocksOfType(PenisType.HORSE) > 0)
+            score++
+        if (hasVagina())
+            /*
+            TODO:vaginas.type is not working. What are we supposed to use?
+            && vaginas.type == VaginaType.EQUINE)
+            */
+            score++
+        if (skin.hasCoatOfType(SkinCoatType.FUR))
+            score++
+            if (arms.type == ArmType.HUMAN)
+                score++
+        if (creature.isTaur
+        /*
+        TODO: Missing races, is this how it should be implemented?
```

Comment:
I'd like to have checks like this to be defined in some racial priority system (unicorn overrides horse, nekomata & cheshire override cat etc), rather than them checking each other's scores. Will decide later as more races get in

Reply:
Okay, for now I'll leave this in comments, other comments will either be removed or changed.